src/modules/person/api/person.controller.ts

```diff
@@ -240,12 +290,27 @@ export class PersonController {
     @ApiInternalServerErrorResponse({ description: 'Internal server error.' })
     @UseInterceptors(ResultInterceptor)
     public async resetPasswordByPersonId(@Param() params: PersonByIdParams): Promise<Result<string>> {
-        const result: Result<string> | SchulConnexError = await this.personUc.resetPassword(params.personId);
+        const person: Option<Person<true>> = await this.personRepository.findById(params.personId);
+        if (!person) {
+            throw SchulConnexErrorMapper.mapSchulConnexErrorToHttpException(
+                SchulConnexErrorMapper.mapDomainErrorToSchulConnexError(
+                    new EntityNotFoundError(`Person with the following ID ${params.personId} does not exist`),
```

Comment:
```suggestion
                    new EntityNotFoundError('Person', params.personId),
```

Might be in more places, maybe do a quick search

Reply:
babe49cd36991f86dfb821a4a422abb07bcfd767